src/BGIter.js

```diff
@@ -0,0 +1,43 @@
+class BGIter {
+  constructor(interval = 1000, iterations = 10) {
+    this.interval = interval;
+    this.iterations = iterations;
+    this.usedColors = {};
+    this.count = 0;
+    this.timeout;
+  }
+
+  static setBg(color) {
+    document.body.style.backgroundColor = color;
+  }
+
+  static genColor() {
+    const red = Math.random() * 255 >> 0;
+    const green = Math.random() * 255 >> 0;
+    const blue = Math.random() * 255 >> 0;
+    return `rgb(${red},${green},${blue})`;
+  }
+
+  run() {
+    this.timer();
+  }
+
+  bgColor() {
```

Comment:
When I see this function for the very first time, it's hard to say exactly what it does just from looking at the name. Clearly, after looking at the code, it generates the color until it finds a unique one. Could you find a better name for this function that reflects what it does?